File: spring-bootstrap-java-audit-swagger/src/main/java/com/marcskow/spring/bootstrap/java/person/model/PersonDto.java
package com.marcskow.spring.bootstrap.java.person.model;

import com.marcskow.spring.bootstrap.java.pet.PetDto;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Set;
import java.util.stream.Collectors;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PersonDto {
    private long id;
    private String firstName;
    private String lastName;
    private int age;
    private long phoneNumber;
    private Set<PetDto> pets;
}

File: spring-bootstrap-complex-criteria-api/src/main/kotlin/com/marcskow/complex/criteria/api/person/model/PersonDto.kt
package com.marcskow.complex.criteria.api.person.model

import com.marcskow.complex.criteria.api.pet.PetDto

data class PersonDto(
        val id: Long = 0,
        val firstName: String,
        val lastName: String,
        val age: Int,
        val phoneNumber: Long,
        val pets: Set<PetDto>
)